src/api/events/registerForEvents.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import type { Auth, ApiResponseSuccess, Narrow } from '../apiTypes';
+import type { InitialData } from '../../types';
+import { apiPost } from '../apiFetch';
+
+/** See https://zulipchat.com/api/register-queue */
```

Comment:
This jsdoc winds up attached to ApiResponseRegister, rather than registerForEvents.

Let's keep jsdoc immediately above what it documents, without a blank line. I think the blank line helps make it easy for something else to get inserted in between, as happens with ApiResponseRegister over the course of this branch.